README.md

```diff
@@ -1,4 +1,14 @@
-# NEAR-SDK-JS
+# NEAR JavaScript SDK
+
+## Quick Start
+
+Use [`create-near-app`](https://github.com/near/create-near-app) to quickly get started writing smart contracts in JavaScript on NEAR.
+
+    npx create-near-app
+
+This will scaffold a basic template for you 😎
+
+Learn more in our [Quick Start guide](https://docs.near.org/develop/quickstart-guide).
```

Comment:
Great! Let's delete two sections below this line, they are not relevant anymore.